server/data/prisonerSearchClient.ts

```diff
@@ -0,0 +1,34 @@
+import config from '../config'
+import createRestClientBuilder from './restClient'
+
+const restClientBuilder = createRestClientBuilder('prisonerSearch', config.apis.prisonerSearch)
+
+export type SearchForm = {
+  prisonNumber?: string
+  firstName?: string
+  lastName?: string
+  agencyId?: string
+}
+
+export type PrisonerSearchClientBuilder = new (token: string) => PrisonerSearchClient
```

Comment:
This type matches the constructor of the client. This allows us to pass the class as an argument in the service:

`new PrisonSearchService(PrisonSearchClient, ... )`

Where the constructor of PrisonSearchService looks like:

` constructor(PrisonerSearchClient: PrisonerSearchClientBuilder,...`

We can also mock it out relatively easy with jest.